spec/searchers/submission_searcher_spec.rb

```diff
@@ -18,8 +18,11 @@
   end
 
   describe '#search_by_contest_id' do
-    let(:params) { acp contest_id: 7 }
+    let(:params) { acp contest_id: 7, membershipable_type: 'Contest' }
```

Comment:
`membershipable_type` is not used in the SubmissionSearcher 

```suggestion
    let(:params) { acp contest_id: 7 }
```